pkg/proto/nbcontract/v1/nodepoolprofile.proto

```diff
@@ -1,9 +0,0 @@
-syntax = "proto3";
-package nbcontract.v1;
-
-message NodePoolProfileConfig {
-
-  // vm_size represents the size of this node pool
-  // Let's keep it for now but continue to review if this can be removed from contract as this can be retrieved from IMDS
-  string vm_size = 1;
```

Comment:
Not seeing this used anywhere for populating cse_cmd, not sure why it was initially added?

Reply:
Added back as it can be used to compute many other contract variables which can then be removed.